joint_names:
  - "wam/base_yaw_joint"
  - "wam/shoulder_pitch_joint"
  - "wam/shoulder_yaw_joint"
  - "wam/elbow_pitch_joint"
  - "wam/wrist_yaw_joint"
  - "wam/wrist_pitch_joint"
  - "wam/palm_yaw_joint"


# max_velocities:    [2.5, 5, 10.0, 4, 24.0, 19.0, 27.0]

# max_velocities:    [3.23, 6.3, 10.0, 10.0, 24.0, 19.0, 27.0]
max_velocities:    [3.9, 6.3, 10.0, 10.0, 24.0, 19.0, 27.0]
max_accelerations: [6.82, 11.5, 17.0, 21.5, 84.0, 110.0, 100.0]

min_positions: [-2.6, -1.98, -2.8, -0.9, -4.65, -1.57, -2.95]
max_positions: [ 2.6,  1.98,  2.8,  3.1,  1.35,  1.57,  2.95]